locale/telemosaic.de.tr

```diff
@@ -0,0 +1,45 @@
+# textdomain: telemosaic
+##[ beacon.lua ]##
+Protected Telemosaic Beacon=Geschützte Telemosaik-Leuchtfeuer
+Telemosaic Beacon=Telemosaik-Leuchtfeuer
+Telemosaic Beacon Active (you hacker you!)=Telemosaik-Leuchtfeuer aktiv (du Hacker du!)
+Protected Telemosaic Beacon Active (you hacker you!)=Geschützte Telemosaik-Leuchtfeuer aktiv (du Hacker du!)
+Telemosaic Beacon Error (you hacker you!)=Telemosaik-Leuchtfeuer fehler (du Hacker du!)
+Protected Telemosaic Beacon Error (you hacker you!)=Geschützte Telemosaik-Leuchtfeuer fehler (du Hacker du!)
+Telemosaic Beacon Disabled (you hacker you!)=Telemosaik-Leuchtfeuer deaktiviert (du Hacker du!)
+Protected Telemosaic Beacon Disabled (you hacker you!)=Geschützte Telemosaik-Leuchtfeuer deaktiviert (du Hacker du!)
+
+##[ extender.lua ]##
+## oder Verlänger, Weiterung?
+Telemosaic Extender, Tier @1 (@2)=Telemosaik-Verstärker Stufe @1 (@2)
+Telemosaic Extender, Tier @1=Telemosaik-Vertstärker Stufe @1
+Grey=Graue
+# Aus MTG Wollen
+White=Weiße
+Dark Grey=Dunkelgraue
+Black=Schwarze
+Violet=Violette
+Blue=Blaue
+Cyan=Türkise
+Dark Green=Dunkgelgrüne
+Green=Grüne
+Yellow=Gelbe
+Brown=Braune
+Orange=Orange
+Red=Rote
+Magenta=Magenta
+Pink=Rosa
```

Comment:
Didn´t test ingame, so those might be wrong depending on how they are used.
For example
> blue eyes = Blaue Augen
> blue car = Blaues Auto
> blue cube = Blauer Würfel

yes german is complicated ;-)

Reply:
All of them are solely used to substitute into the string `Telemosaic Extender, Tier @1 (@2)=Telemosaik-Verstärker Stufe @1 (@2)`. For example `Telemosaik-Verstärker Stufe 1 (Rote)`.

I had assumed these were the correct declensions for Telemosaic Extenders the same as "Red Wool", "Green Wool" and so on from Minetest Game's `wool`, though I'm noticing now `Wolle` is feminine while `Verstärker` is masculine.

Reply:
They are correct for the wool mod because they are used as adjectives describing the color, more like `reddish wool`.

In telemosaics case inside `( )` its a characteristic and just the color name is used `Rot/Grün/Blau/etc`.